### ISSUE TITLE
OSX Install - Error: No Brewfile found

### ISSUE THREAD

**Issue by hasen6** [HaxeFoundation/hashlink#348]
`hasen$ brew bundle
==> Tapping homebrew/bundle
Cloning into '/usr/local/Homebrew/Library/Taps/homebrew/homebrew-bundle'...
remote: Enumerating objects: 92, done.
remote: Counting objects: 100% (92/92), done.
remote: Compressing objects: 100% (82/82), done.
remote: Total 92 (delta 5), reused 29 (delta 4), pack-reused 0
Unpacking objects: 100% (92/92), done.
Checking connectivity... done.
Tapped 1 command (188 files, 264.4KB).
Error: No Brewfile found
hasen$ make
make: *** No targets specified and no makefile found.  Stop.`

**Comment by Aurel300** [comment]
Are you running `brew bundle` in the correct directory? The `Brewfile` is in the root of the hashlink repository.

**Comment by hasen6** [comment]
I just followed the instructions here: https://github.com/HaxeFoundation/hashlink#building-on-linuxosx

It just says run `brew bundle` nothing about which directory to run it in.

**Comment by Aurel300** [comment]
I'd say it is implicit. We can add it to the instructions, but please confirm it works if you run in the correct directory (`cd` into the cloned repository before running the commands).

**Comment by hasen6** [comment]
Where is it implied? Not sure we're looking at the same page. I've used brew many times in the past and the commands were always executed from the home directory.

Literally all it says is "`brew bundle` to install the dependencies listed in Brewfile."  I would say the word 'execute this command in terminal' is implied is all.

Please let me know which directory I need to run `brew bundle` from. The only brewfile I found in `usr/local` is in `usr/local/Homebrew/Library/Taps/homebrew/homebrew-bundle/lib/bundle/`

**Comment by Aurel300** [comment]
Yes, when you install dependencies by name with `brew install <xyz>`, it does not matter where it is run from. But, when using `brew bundle` (from the [`README`](https://github.com/Homebrew/homebrew-bundle)):

> `brew bundle` will look for `Brewfile` in the current directory.

The directory you need to run `brew bundle` in is the cloned repository `hashlink`. If you did something like `git clone git@github.com:HaxeFoundation/hashlink.git`, then this creates a `hashlink` directory. `cd` into it, then run the commands.

**Comment by hasen6** [comment]
> Yes, when you install dependencies by name with brew install <xyz>, it does not matter where it is run from. But, when using brew bundle (from the README):

Ok if you know `brew bundle` is very different but I'm used to running `brew something` and it's always been from anywhere.

I've been following this https://heaps.io/documentation/installation.html to install everything required for heaps.io. I installed Haxe already from the installer, then I already had vscode...then all it says is for OSX go to your install page and that tells me to do `brew bundle`.

So those are the only steps I've taken so far.
